flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/common/JoinReorderTestBase.scala

```diff
@@ -121,6 +121,8 @@ abstract class JoinReorderTestBase extends TableTestBase {
 
     util.getTableEnv.getConfig
       .set(OptimizerConfigOptions.TABLE_OPTIMIZER_JOIN_REORDER_ENABLED, Boolean.box(true))
+    util.getTableEnv.getConfig
```

Comment:
we should rename `JoinReorderTestBase` to LeftDeepJoinReorderTestBase

Reply:
> we should rename `JoinReorderTestBase` to LeftDeepJoinReorderTestBase

Done!